### ISSUE TITLE
tweak to the logic to better support deposit of bronze materials.

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1756]
Duplicate of `https://github.com/OAButton/discussion/issues/1516` & https://github.com/OAButton/discussion/issues/1733 that I'm splitting out into it's own thing. 

At the moment, when papers are `bronze` we are providing a `redeposit` option e.g `https://shareyourpaper.org/10.1016/j.jelekin.2019.102379`. This is correct as I specified it, but I made a mistake. Since re-use rights aren't associated.

Instead, we should provide an option for file deposit in these cases (in line with what our the new permissions API says).

The UI needs slightly different copy to deal with the fact it's already free, and I've tried to add that into the code with the ID `_oab_bronze_archivable`. 🤞 if you uncomment that it'll work.

**Comment by JosephMcArthur** [comment]
@markmacgillivray, after and only after we ship the refactor live would you be able to look at this please?

**Comment by markmacgillivray** [comment]
Can this be done after #1842 or do we have to do it before? Otherwise we'll just be adding to the list of things we're doing in two places.

**Comment by JosephMcArthur** [comment]
After is better. Let's unassign for now as we should try and keep that a
clear refactor too.
